### DISCUSSION TITLE
Ui [0.14] yellow bar

### DISCUSSION THREAD

**Discussion by phlnil** [blakeblackshear/frigate#13439] [Ask A Question] [answered]
During a review (0.14) on the left of the screen  is the width of the bar directly proportional to some value? Sometimes it seems to be based on how much the pixels have changed from previous and subsequent frames... but sometimes not... either way it is useful (obviously keep this mine as a question and not a criticism !). 

Thank you.

**Comment by NickM-27** [comment, marked as answer]
It represents the level of motion during that time period within the hour. so for example if it was raining on your camera, it would not show as a high level of motion the whole hour because that is not useful, the widest yellow bar would be the time where motion was highest within that hour. 

you can't create a direct proportionality because then a dog for example would never show up on the timeline since the amount of pixels that make up a dog is insignificant compared to that of a car.

**Comment by phlnil** [comment]
Thank you as always!
